### DISCUSSION TITLE
Does this support KMP?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [evant/kotlin-inject#283] [Q&A] [answered]
Working on a greenfield KMP project soon and I'm curious if this supports multiplatform as id like to use it. didn't see anything in the readme or issues.

**Comment by tfcporciuncula** [comment, marked as answer]
It does indeed! There are two multiplatform samples [here](https://github.com/evant/kotlin-inject-samples/tree/main/multiplatform).

**Reply by ColtonIdle** [reply]
No way! Awesome!
